### ISSUE TITLE
[tobi] Cannot see focused elements on Product Carousel

### ISSUE THREAD

**Issue by Zeyber** [SAP/cloud-commerce-spartacus-storefront#4674]
When tabbing through product carousel, we cannot see the products that are focused off the screen. The carousel should switch to the next/previous slide so that we can see the focused product.

**Comment by kacperknapik** [comment]
during fixing visibility, we can add arrow keys functionality (to go through the products in carousel) note: add that to auxiliary keys tests also

**Comment by kacperknapik** [comment]
currently @tobi-or-not-tobi is taking care of this fix. he mentioned that this was already implemented, but is not working. he will investigate

**Comment by developpeurweb** [comment]
Issue seems unrelated to _visible focus_ in a strict sense, since -as per screenshot- the elements that are visible do have _visible focus_, therefore moving to **BACKLOG**.

![Screen Shot 2020-01-31 at 9 55 53 PM](https://user-images.githubusercontent.com/15113219/73585980-42c09080-4475-11ea-8755-76384bba2659.png)

**Comment by kacperknapik** [comment]
This will be fixed in another ticket (not yet created)
